plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/containers/N4JSToBeBuiltComputer.java

```diff
@@ -60,28 +55,9 @@
 	@Inject
 	private UriValidator uriValidator;
 
-	/**
-	 * Caches the URIs that are contained in an archive such that the archive is not opened each and every time the
-	 * contents has to be iterated.
-	 *
-	 * This one is used in addition to the cache in the {@link NfarStorageMapper} because it's used in a different
-	 * life-cycle, e.g. the builder is triggered asynchronously and has to provide information about files after they
-	 * have been removed physically from disk. The {@link NfarStorageMapper} updates its cache synchronously to the
-	 * delete event.
-	 */
-	private final Map<String, Set<URI>> knownEntries = Maps.newConcurrentMap();
-
 	@Override
 	public void removeProject(ToBeBuilt toBeBuilt, IProject project, IProgressMonitor monitor) {
-		storageMapper.collectNfarURIs(project, toBeBuilt.getToBeDeleted());
-		Iterator<Map.Entry<String, Set<URI>>> iterator = knownEntries.entrySet().iterator();
-		String keyPrefix = project.getName() + "|";
-		while (iterator.hasNext()) {
-			String key = iterator.next().getKey();
-			if (key.startsWith(keyPrefix)) {
-				iterator.remove();
-			}
-		}
+		// nothing to remove
```

Comment:
do we even need this method, then? (same for `#removeStorage()` below)

Reply:
It is an interface implementation, thus we need the stubs (cf. `IToBeBuiltComputerContribution`).